File: movieApplication 5/src/app/Movie.ts
export class Movie{
    constructor(
        public movieId: number,
        public overView: String,
        public popularity: number,
        public releaseDate: String,
        public title: String,
        public voteAverage: number,
        public posterPath: String,
        public watched: boolean 
    ){}
}

File: MovieService 2/FavouriteMovieService/src/main/java/com/movie/FavouriteMovieService/model/Movie.java
package com.movie.FavouriteMovieService.model;

import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;

@Data
@NoArgsConstructor
public class Movie {

    @Id
    int movieId;
    String overView;
    float popularity;
    String releaseDate;
    String title;
    float voteAverage;
    String posterPath;
    boolean watched;

}
